genson/src/main/java/com/owlike/genson/ext/jackson/JacksonBundle.java

```diff
@@ -18,61 +28,177 @@
 public class JacksonBundle extends GensonBundle {
     @Override
     public void configure(GensonBuilder builder) {
-        builder
-              .with(new JacksonAnnotationsResolver())
-              .with(new JacksonNameResolver());
+        builder.with(new JacksonAnnotationsResolver())
+            .with(new JacksonNameResolver());
     }
 
     private static class JacksonAnnotationsResolver implements BeanMutatorAccessorResolver {
         @Override
         public Trilean isCreator(Constructor<?> constructor, Class<?> fromClass) {
-            return null;
+            if (find(JsonCreator.class, constructor, fromClass) != null) {
+                return Trilean.TRUE;
+            }
+            return Trilean.UNKNOWN;
         }
 
         @Override
         public Trilean isCreator(Method method, Class<?> fromClass) {
-            return null;
+            if (find(JsonCreator.class, method, fromClass) != null) {
+                return Trilean.TRUE;
+            }
+            return Trilean.UNKNOWN;
         }
 
         @Override
         public Trilean isAccessor(Field field, Class<?> fromClass) {
-            return null;
+            if (ignore(field, field.getType())) {
+                return Trilean.FALSE;
+            }
+            if (include(field, field.getType())) {
+                return Trilean.TRUE;
+            }
+            return Trilean.FALSE;
```

Comment:
Shouldn't this be `Trilean.UNKNOWN`, to allow other resolvers to kick in?

I believe if you return either `TRUE` or `FALSE` Genson considers it a "final word" and doesn't consult other resolvers in the chain, but if you return `null` or `UNKNOWN` it does.

We could have a field that doesn't have Jackson annotation, but it does have an equivalent Genson, JAXB, or JSON-B annotation. We need to make sure that will work, regardless of the order of resolvers.